### ISSUE TITLE
Beta UI: button for macros is not labeled

### ISSUE THREAD

**Issue by klml** [zammad/zammad#6165]
### Used Zammad Version

7.0.1-df77241a.docker

### Environment

openshift

### Actual behaviour

The “^” button for macros is not labeled:

<img width="63" height="61" alt="Image" src="https://github.com/user-attachments/assets/59b36031-ca5c-40ca-b765-305972917386" />

### Expected behaviour

The “^” button for macros should be labeled.

### Steps to reproduce the behaviour

Use Screenreader

### Support Ticket

Ticket#10226081

### I'm sure this is a bug and no feature request or a general question.

no
